### DISCUSSION TITLE
[Unstable] Calyxanide (and poisons) does almost zero damage to husks

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14671] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Husks are extremely resilient to Calyxanide and other poisons (all causing organ damage) despite calyxanide/poisons being used as potential treatments against husk infection. 
Coupled with the constant health regeneration of husks, damage over time is very weak.

Husks need an organ damage multiplier which would make them more vulnerably to poisons and Calyxanide needs a potential damage buff as well (it's an expensive med it should be strong)
I can't think of any "conventional" weapons doing direct organ damage.

1x Calyxanide on basic husk (constantly healing)
![image](https://github.com/user-attachments/assets/5018d4fe-967c-4497-bb84-d305d73378dc)

16x !!!!! Calyxanide on husk exosuit
![image](https://github.com/user-attachments/assets/d8b6519a-6add-49bf-920c-c4058e3cb618)

### Reproduction steps

Try killing any husk with calyxanide I dare you

### Bug prevalence

Just once

### Single player or multiplayer?

Other

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Calyxanide now can knock out most of the husk variants and bigger ones need 2.
Closing discussion as resolved.
